packages/SwingSet/src/kernel/kernel.js

```diff
@@ -64,11 +64,19 @@ export default function buildKernel(kernelEndowments) {
     parseKernelSlot(target);
     insistMessage(msg);
     const m = harden({ type: 'send', target, msg });
+    kernelKeeper.incrementKernelPromiseRefCount(target, `enq|msg|t`);
```

Comment:
It looks like we're sending both object and promise refs into this function, but it ignores everything except promises. Maybe the function should simply be named `incrementRefCount`? We could argue that we don't manage object refcounts yet (but we will, eventually, along with device nodes), and I don't see anything here that would have to change when we make that improvement. And then it doesn't look so funny to be passing non-Promises into the function.